pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerGauge.java

```diff
@@ -45,8 +45,10 @@ public enum ServerGauge implements AbstractMetrics.Gauge {
   // Dedup metrics
   DEDUP_PRIMARY_KEYS_COUNT("dedupPrimaryKeysCount", false),
   CONSUMPTION_QUOTA_UTILIZATION("ratio", false),
-  JVM_HEAP_USED_BYTES("bytes", true);
-
+  JVM_HEAP_USED_BYTES("bytes", true),
+  // Lag metrics
+  MAX_CONSUMPTION_DELAY_MS("milliseconds", false),
```

Comment:
rename this as `TABLE_MAX_CONSUMPTION_DELAY_MS` or `TABLE_MAX_INGESTION_DELAY_MS` ?

Reply:
Yes, makes sense to add the TABLE prefix to both and settle on ingestion delay naming everywhere